src/plans/details/tabs/Mappings/PlanMappingsPage.tsx

```diff
@@ -47,30 +117,45 @@ const PlanMappingsPage: FC<PlanPageProps> = ({ name, namespace }) => {
 
   return (
     <PageSection hasBodyWrapper={false} data-testid="plan-mappings-section">
-      <MappingAlerts
-        alerts={[
-          ...getMappingAlerts({
-            resourcesError,
-            sourceNetworkEmpty: isEmpty(sourceNetworks),
-            sourceStoragesEmpty: isEmpty(sourceStorages),
-            targetStoragesEmpty: isEmpty(targetStorages),
-          }),
-          alert,
-        ].filter(Boolean)}
-      />
       <SectionHeading text={t('Mappings')} />
 
-      <PlanMappingsSection
-        plan={plan}
-        sourceProviderType={sourceProviderType}
-        planNetworkMap={planNetworkMap!}
-        planStorageMap={planStorageMap!}
-        setAlertMessage={setAlert}
-        sourceNetworks={sourceNetworks}
-        sourceStorages={sourceStorages}
-        targetNetworks={targetNetworks}
-        targetStorages={targetStorages}
+      <SectionHeadingWithEdit
```

Comment:
Worth adding a link ot a string of the created mappings entity name for both network and storage, the same as was in old UI. Otherwise it will be hard for the user to find this mapping in the network, storage maps list page.

Reply:
I added it to the UI, this is the best location I could think of, @mmenestr can you have an opinion on this
<img width="811" height="588" alt="Screenshot - 2025-12-17T211300 059" src="https://github.com/user-attachments/assets/1eac7901-eb2d-4701-9a6e-48b070c9251b" />

Reply:
Editing names of resources in kubernetes like network maps/plan/providers is not allowed, but I will change the display accordingly, Thank you